lib/rules/jsx-no-target-blank.js

```diff
@@ -265,11 +266,11 @@ module.exports = {
             return;
           }
 
-          const formAttribute = formComponents.get(node.name.name);
+          const formAttributes = formComponents.get(node.name.name);
 
           if (
-            hasExternalLink(node, formAttribute)
-            || (enforceDynamicLinks === 'always' && hasDynamicLink(node, formAttribute))
+            hasExternalLink(node, formAttributes)
+            || (enforceDynamicLinks === 'always' && hasDynamicLink(node, formAttributes))
```

Comment:
seems like we need code coverage for this line; any chance you could add it?

Reply:
Am planning to but I'm confused as I kinda think the test case for it is already there... Will take another look later.

Reply:
@ljharb should be good now